lib/configuration.js

```diff
@@ -9,9 +9,13 @@ const CONFIGURATION_FILES = [
   '.versionrc.js',
 ];
 
-module.exports.getConfiguration = function () {
+module.exports.getConfiguration = function (argv) {
```

Comment:
Optional: it feels a bit cleaner to pass in the config filename rather than the whole `argv` here, but I won't block merge over it.